pages/myinfo/notification/subscribe/components/SubscribeCard.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import Image from 'next/image';
+
+import { MainButtonV2 } from '@components/common/buttons/mainButtonsV2';
+
+import { useMediaQueryContext } from '@/contexts/MediaQueryContext';
+
+export default function SubscribeCard({
+  logoImage,
+  company,
+  isSubscribe,
+}: {
+  logoImage: string;
+  company: string;
+  isSubscribe: boolean;
+}) {
+  const { isMobile } = useMediaQueryContext();
+
+  return (
+    <div
+      className={`border-gray600 flex flex-col ${isMobile ? 'min-w-[16.35rem]' : 'min-w-[18.025rem]'}`}
+    >
+      <div
+        className={`rounded-t-Radius16 bg-gray700 flex items-center justify-center h-[8rem] ${isMobile ? 'p-[3.175rem]' : 'p-[4.013rem]'}`}
+      >
+        <Image src={logoImage} alt='회사 로고' width={100} height={100} />
+      </div>
+
+      <div className='rounded-b-Radius16 border border-gray600 border-t-0 p-[1.6rem] flex flex-col gap-[1.6rem] justify-center text-center'>
+        <b className='st2 font-bold'>{company}</b>
+        <div className='flex flex-col gap-[0.8rem]'>
+          <MainButtonV2 text='아티클 보기' color='gray' size='small' radius='square' line={false} />
+          {isSubscribe ? (
```

Comment:
```suggestion
        <MainButtonV2
            text={isSubscribe ? '구독 중' : '구독하기'}
            color='primary'
            line={isSubscribe}
            {...buttonProps}
          />
```


```
const buttonProps = {
    size: 'small',
    radius: 'square',
  };
  ```
  
  이런식으로도 처리 할 수 있을것같네요!

Reply:
옹 좋아요! 수정했습니댜!
[refactor: SubscribeCard 구독 버튼 리팩토링](https://github.com/dreamyPatisiel/devdevdev-client/pull/249/commits/0752d9d0d559bd35a9a994ce594cf2c0652cd5e4)